src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -14,13 +14,12 @@ namespace Microsoft.DocAsCode.Build.Engine
     using HtmlAgilityPack;
 
     using Microsoft.DocAsCode.Common;
-    using Microsoft.DocAsCode.Dfm;
     using Microsoft.DocAsCode.MarkdownLite;
     using Microsoft.DocAsCode.Plugins;
     using Microsoft.DocAsCode.Utility;
 
     [Export(typeof(IHostService))]
-    internal sealed class HostService : IHostService, IDisposable
+    public sealed class HostService : IHostService, IDisposable
```

Comment:
MarkdownReaderTest need HostService instance to do markup...

---
In reply to: [65490284](https://github.com/dotnet/docfx/pull/343#discussion_r65490284) [](ancestors = 65490284)